uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/contact-actions-menu.json

```diff
@@ -2,13 +2,13 @@
   "scenarios": [
     {
       "label": "Contact Summary - Action Menu",
-      "url": "index.php?q=civicrm/contact/view&reset=1&cid={{contactId:staff}}",
```

Comment:
I guess you were just testing, could you put the tags back please?

Reply:
Ops! sorry. Adding back!